doc/source/cluster/kubernetes/user-guides/rayserve-dev-doc.md

```diff
@@ -50,7 +50,7 @@ the second `mobilenet` is the name of the Python file in the directory `mobilene
 helm uninstall raycluster
 
 # Install the RayCluster CR with the Ray image `rayproject/ray-ml:${RAY_VERSION}`
-helm install raycluster kuberay/ray-cluster --version 1.3.0 --set image.repository=rayproject/ray-ml
```

Comment:
We should not do this. It may cause differences in the Ray version.

Reply:
How about removing Step 5 and using runtime env + UV (you should use KubeRay v1.4 RayCluster Helm chart) instead?